e2e-polybft/cardanofw/test-cardano-token-tx.go

```diff
@@ -65,6 +65,54 @@ func FundUserWithToken(ctx context.Context, chain ChainID,
 	return &fundToken, nil
 }
 
+func FundAddressWithToken(ctx context.Context, chain ChainID,
+	networkType cardanowallet.CardanoNetworkType, txProvider cardanowallet.ITxProvider,
+	minterUser *TestApexUser, addrToFund string, lovelaceFundAmount uint64, tokenFundAmount uint64,
+) (*cardanowallet.TokenAmount, error) {
+	minterWallet, _ := minterUser.GetCardanoWallet(chain)
```

Comment:
duplicate code from `GetTokenName`
change that to receive wallet and than use here also